content/sensu-go/5.0/api/checks.md

```diff
@@ -167,7 +169,7 @@ example payload | {{< highlight shell >}}
   }
 }
 {{< /highlight >}}
-payload parameters | Required check attributes: `command` (string), `subscriptions` (array of strings), `interval` (integer) OR `cron` (string), `publish` (set to `true`), and a `metadata` scope containing `name` (string) and `namespace` (string). For more information about creating checks, see the [check reference][1]. 
+payload parameters | Required check attributes: `interval` (integer) or `cron` (string), and a `metadata` scope containing `name` (string) and `namespace` (string). For more information about creating checks, see the [check reference][1]. 
```

Comment:
Thanks for updating this with the current behavior. I filed https://github.com/sensu/sensu-go/issues/2551, because this is indeed a bug!